benchmark_runner/workloads/bootstorm_vm.py

```diff
@@ -207,8 +207,8 @@ def _verify_vm_ssh(self):
             if self._wait_for_upgrade_version:
                 logger.info(f'Cluster is upgraded to: {self._wait_for_upgrade_version}')
             if failure:
-                self._oc.generate_cnv_must_gather(destination_path=self._run_artifacts_path, cnv_version=self._cnv_version)
-                self._oc.generate_odf_must_gather(destination_path=self._run_artifacts_path, odf_version=self._odf_version)
+                self._oc.generate_cnv_must_gather(destination_path=self._run_artifacts_path)
+                self._oc.generate_odf_must_gather(destination_path=self._run_artifacts_path)
```

Comment:
What is the reason for this change?

Reply:
We dont know when it will be failed before upgrade or after, so we dont know the cnv/odf version in advance

Reply:
OK